### PR TITLE
Hash each part independently

### PR DESCRIPTION
**Pull request by wmbest2**
@vokal/android-developers

### CODE REVIEW

**Comment by DavidRyan** [comment]
lgtm

File: src/main/java/io/vokal/mocktrofit/MockClient.java [hunk 1]
```diff
@@ -216,7 +257,9 @@ public Response execute(Request request) throws IOException {
             output = serve(fullPath);
         } else {
             Log.d("Mocktrofit", "Missing File: " + filename);
-            Log.d("Mocktrofit", "    Key: " + key);
+            Log.d("Mocktrofit", "    File: " + encryptBody(filename));
+            Log.d("Mocktrofit", "    File: " + encryptPathAndBody(filename));
```
Comment (nick-fedesna) [inline src/main/java/io/vokal/mocktrofit/MockClient.java]: might want to clarify these logs

Comment (wmbest2) [inline src/main/java/io/vokal/mocktrofit/MockClient.java]: any idea what you'd like to see there instead?

Comment (nick-fedesna) [inline src/main/java/io/vokal/mocktrofit/MockClient.java]: just not File twice:  Body:, Path+Body: perhaps

**Pull request merged**